builtin/providers/google/resource_compute_instance_group.go

```diff
@@ -0,0 +1,301 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceComputeInstanceGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeInstanceGroupCreate,
+		Read:   resourceComputeInstanceGroupRead,
+		Update: resourceComputeInstanceGroupUpdate,
+		Delete: resourceComputeInstanceGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"fingerprint": &schema.Schema{
```

Comment:
I've the removed `fingerprint` as suggested it was unnecessary. 

In terms of the 3 way merge would you suggest I just call `resourceComputeInstanceGroupRead` before to make sure no other changes have occurred? Or can I assume the state to be correct?